.github/workflows/kubeflow-release.yaml

```diff
@@ -2,7 +2,13 @@
 name: Kubeflow Release Pipeline
 on:
   workflow_dispatch:
+    inputs:
+      checkout-new-release:
+        description: "Checkout a new release?"
```

Comment:
Is "checkout" the right world? why not "create"?

Reply:
right, I will change to create